go.mod

```diff
@@ -4,7 +4,9 @@ go 1.24
 
 require (
 	github.com/mark3labs/mcp-go v0.26.0
```

Comment:
Did you run go mod vendor? 
I can see changes in go.mod file, but nothing is commited to vendor folder

Reply:
Yes, there are no changes to vendor